Test/FMI3/fmi3_import_interface_types_test.cpp

```diff
@@ -0,0 +1,129 @@
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_me_providesDirectionalDerivatives)      == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_me_providesAdjointDerivatives)          == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_me_providesPerElementDependencies)      == 1);
+        // ME
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_me_needsCompletedIntegratorStep)   == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_me_providesEvaluateDiscreteStates) == 1);
+    }
+    SECTION("CS") {
+        REQUIRE_STREQ(fmi3_import_get_model_identifier_CS(fmu), "id_cs");
+        // Common
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_cs_needsExecutionTool)                  == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_cs_canBeInstantiatedOnlyOncePerProcess) == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_cs_canGetAndSetFMUState)                == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_cs_canSerializeFMUState)                == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_cs_providesDirectionalDerivatives)      == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_cs_providesAdjointDerivatives)          == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_cs_providesPerElementDependencies)      == 1);
+        // CS
+        REQUIRE(     fmi3_import_get_capability(fmu, fmi3_cs_canHandleVariableCommunicationStepSize) == 1);
+        REQUIRE(fabs(fmi3_import_get_cs_fixed_internal_step_size(fmu) - 3.0)                         <  0.00001);
+        REQUIRE(     fmi3_import_get_capability(fmu, fmi3_cs_maxOutputDerivativeOrder)               == 5);
+        REQUIRE(     fmi3_import_get_cs_recommended_intermediate_input_smoothness(fmu)               == -2);
+        REQUIRE(     fmi3_import_get_capability(fmu, fmi3_cs_providesIntermediateUpdate)             == 1);
+        REQUIRE(     fmi3_import_get_capability(fmu, fmi3_cs_mightReturnEarlyFromDoStep)             == 1);
+        REQUIRE(     fmi3_import_get_capability(fmu, fmi3_cs_canReturnEarlyAfterIntermediateUpdate)  == 1);
+        REQUIRE(     fmi3_import_get_capability(fmu, fmi3_cs_hasEventMode)                           == 1);
+        REQUIRE(     fmi3_import_get_capability(fmu, fmi3_cs_providesEvaluateDiscreteStates)         == 1);
+    }
+    SECTION("SE") {
+        REQUIRE_STREQ(fmi3_import_get_model_identifier_SE(fmu), "id_se");
+        // Common
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_se_needsExecutionTool)                  == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_se_canBeInstantiatedOnlyOncePerProcess) == 1);
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_se_canGetAndSetFMUState)                == 1);
+        /* Have at least one value off by design */
+        REQUIRE(fmi3_import_get_capability(fmu, fmi3_se_canSerializeFMUState)                == 0);
```

Comment:
Note that it was intentional to set it to `0`, I was thinking there could be a slight chance we are reading from the wrong memory or maybe even cached values from the ME or CS tests above.